Plugins/NativeEngine/Source/ShaderCompilerTraversers.cpp

```diff
@@ -451,8 +451,21 @@ namespace Babylon::ShaderCompilerTraversers
 #define IF_NAME_RETURN_ATTRIB(varyingName, attrib, newName)  \
     if (std::strcmp(name, varyingName) == 0)                 \
     {                                                        \
-        return {static_cast<unsigned int>(attrib), newName}; \
+        return {static_cast<unsigned int>(attrib), name}; \
     }
+#if ANDROID
```

Comment:
Is this change only for Android or for OpenGL?

Reply:
This should be for OpenGL, not just android. I just used this preprocessor flag to test. How should we branch based off the current renderer?

Reply:
Changed `ANDROID` preprocessor to `APIOpenGL`, that should be appropriate, right?

Reply:
yep!